src/TestStack.Seleno/Configuration/AppConfigurator.cs

```diff
@@ -52,7 +51,7 @@ public ISelenoApplication CreateApplication()
 
         private IContainer BuildContainer()
         {
-            ContainerBuilder.Register(c => new ProxyGenerator(disableSignedModule: true))
+            ContainerBuilder.Register(c => new ProxyGenerator(true))
```

Comment:
I prefer the label here - passing in a random boolean isn't very intention revealing